script/Base.s.sol

```diff
@@ -38,4 +44,25 @@ abstract contract BaseScript is Script {
         _;
         vm.stopBroadcast();
     }
+
+    /// @dev The presence of the salt instructs Forge to deploy contracts via this deterministic CREATE2 factory:
+    /// https://github.com/Arachnid/deterministic-deployment-proxy
+    ///
+    /// Notes:
+    /// - The salt format is "ChainID <chainid>, Version <version>".
+    /// - The version is obtained from the `package.json` file via --ffi cheatcode:
+    /// https://book.getfoundry.sh/cheatcodes/ffi
```

Comment:
Let's add another note before this that says that the user needs to have the `jq` CLI tool installed.